src/ui/public/notify/__tests__/notifier.js

```diff
@@ -401,3 +401,159 @@ describe('Notifier', function () {
     });
   }
 });
+
+describe('Directive Notification', function () {
+  let notifier;
+  let compile;
+  let scope;
+
+  const directiveParam = {
+    template: '<h1>Hello world {{ unit.message }}</h1>',
+    controllerAs: 'unit',
+    controller() {
+      this.message = '🎉';
+    }
+  };
+  const customParams = {
+    title: 'fooTitle',
+    actions:[{
+      text: 'Cancel',
+      callback: sinon.spy()
+    }, {
+      text: 'OK',
+      callback: sinon.spy()
+    }]
+  };
+  let directiveNotification;
+
+  beforeEach(() => {
+    ngMock.module('kibana');
+    ngMock.inject(function ($rootScope, $compile) {
+      scope = $rootScope;
+      compile = $compile;
+      compile;
+      scope;
```

Comment:
Right sorry, this was just to get rid of some unused variable warnings. This PR is still in-progress until I can come up with a test that covers `notif.directive.link` at least